components/tools/OmeroPy/src/omero/plugins/web.py

```diff
@@ -59,6 +59,9 @@ def config_required(func):
     """Decorator validating Django dependencies and omeroweb/settings.py"""
     def import_django_settings(func):
         def wrapper(self, *args, **kwargs):
+            if self._isWindows():
+                self.ctx.err("Windows platform will not longer be supported"
```

Comment:
Perhaps: "Support for Windows will be removed in OMERO 5.3"? Provide http://blog.openmicroscopy.org/tech-issues/future-plans/deployment/2016/03/22/windows-support/ (or too long)?